specs/latest/2.0/index.html

```diff
@@ -3443,6 +3443,22 @@ <h3>Preventing undefined behavior with Transform Feedback</h3>
         OpenGL ES 3.0.5 &sect;2.15.2</a>)</span>
     </div>
 
+    <p>
+        In case the same buffer is bound to more than one transform feedback binding point that will
+        be written to, and the buffer has been bound with <code>bindBufferBase</code> or the ranges
+        in the buffer chosen by <code>bindBufferRange</code> overlap, <code>BeginTransformFeedback</code>
+        generates an <code>INVALID_OPERATION</code> error.
+    </p>
+
+    <div class="note">
+        This can happen only in <code>SEPARATE_ATTRIBS</code> mode.
+    </div>
+
+    <div class="note rationale">
+        Behavior when the same buffer is bound to two outputs is not specified in the OpenGL ES spec
```

Comment:
Instead of "same buffer" should we write "same region of the same buffer"?

Reply:
Change the rationale to:
This is a restriction from D3D11, where this behavior is not allowed.